griptape/drivers/observability/base_observability_driver.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from typing import TYPE_CHECKING, Any, Optional
+
+from attrs import define
+
+if TYPE_CHECKING:
+    from types import TracebackType
+
+    from griptape.common import Observable
+
+
+@define
+class BaseObservabilityDriver(ABC):
+    def __enter__(self) -> None:  # noqa: B027
+        pass
```

Comment:
Should BaseObservabilityDriver raise a `NotImplementedError` for __enter___ and __exit__?

Reply:
I felt like the setup/teardown functionality of the observability driver is more like opt-in, so rather than requiring subclasses to add this default, no-op implementation, I opted to put it in the base class. I had this same conversation with @collindutter previously.